packages/cli/src/ui/components/messages/__snapshots__/ToolGroupMessage.test.tsx.snap

```diff
@@ -99,21 +99,21 @@ exports[`<ToolGroupMessage /> > Golden Snapshots > renders single successful too
 `;
 
 exports[`<ToolGroupMessage /> > Golden Snapshots > renders sticky header when scrolled 1`] = `
```

Comment:
this snapshot was ugly. Fixed to actually match the case we were trying to test.